.github/workflows/clp-artifact-build.yaml

```
@@ -649,16 +678,35 @@ jobs:
             )}}
 
   package-image:
-    name: "package-image"
+    name: "package-image-${{matrix.arch}}"
     if: >-
       !cancelled() && !failure() && (
         needs.filter-relevant-changes.outputs.ubuntu_jammy_image_changed == 'false' ||
```

actionlint (1.7.12)

[error] got unexpected character '#' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)